src/service/faas.service.ts

```diff
@@ -354,6 +380,39 @@ export class FaasService implements IFaaSService {
     return this.doFetch({ urlPart: '/events', method: 'GET' });
   }
 
+  public async getLogs({
+    uuid,
+    start,
+    end,
+    levels,
+    removeHeader,
+  }: {
+    uuid: string;
+    start?: number;
+    end?: number;
+    levels?: string[];
+    removeHeader?: boolean;
+  }): Promise<void> {
+    const urlPart = '/logs/export';
+    let additionalParams = `&lambdaUUID=${uuid}&startTimestamp=${start}&endTimestamp=${
+      end || Date.now()
+    }`;
+
+    if (levels && levels.length > 0) {
+      for (const level of levels) {
+        additionalParams += `&filterLevels=${level}`;
+      }
+    }
+
+    return this.getStream(
```

Comment:
await the end of the stream so in the meantime a "Downloading lambda logs" hint can be displayed.

Reply:
Decision: Won't be implemented (because of redirect to file)